torch_xla/csrc/xla_lower_util.cpp

```diff
@@ -135,4 +135,20 @@ xla::XlaOp BuildDropout(const xla::XlaOp& input, float probability) {
   return input * mask;
 }
 
+std::vector<xla::XlaOp> CreateBroadcastTensors(
+    tensorflow::gtl::ArraySlice<const xla::XlaOp> operands) {
+  XLA_CHECK(!operands.empty()) << "broadcast_tensors cannot take an empty list";
+  xla::Shape result_shape = XlaHelpers::ShapeOfXlaOp(operands.front());
```

Comment:
Are we sure these are the semantics?

Reply:
As far as I understand it, the broadcast semantics is the same as for elementwise operations, for which we already use this semantics. Also, the documentation only describes it between two tensors; this is merely applying left associativity to it. I can't think of any other semantics which would work.